cognition_objects/environment_variable.py

```diff
@@ -76,12 +130,25 @@ def get_all_by_project_id(project_id: str) -> List[CognitionEnvironmentVariable]
 
 
 def get_cognition_project_env_var_value(cognition_project_id: str) -> str:
+
+    env_var_id = cast(CognitionProject.llm_config.op("->>")("envVarId"), UUID)
     v = (
         session.query(CognitionEnvironmentVariable.value)
-        .join(
-            CognitionProject,
-            CognitionProject.open_ai_env_var_id == CognitionEnvironmentVariable.id,
+        .join(CognitionProject, env_var_id == CognitionEnvironmentVariable.id)
+        .filter(
+            CognitionProject.id == cognition_project_id,
         )
+        .first()
+    )
+    if v:
```

Comment:
i think v can also exists if the result is NULL so i usually check `v and v[0]` to prevent str(None) = 'None'